rust/connlib/tunnel/src/client.rs

```diff
@@ -379,7 +379,7 @@ impl ClientState {
                         }
                     })
                     .unwrap_or_else(|e| {
-                        tracing::debug!(error = std_dyn_err(&e), "Recursive UDP DNS query failed");
+                        tracing::warn!(error = std_dyn_err(&e), "Recursive UDP DNS query failed");
```

Comment:
These are tricky. I kind of want to know when these are happening excessively but the occasional failure here is fine. Although each failure could technically be user-noticable 🤔